oaknational/Oak-Web-Application
# feat: legacy filter + cohort for legacy search hits - LESQ-519

## Description

- Sends legacy string param to search api for filtering out legacy or filtering out legacy but showing eyfs 
- Used cohort field for legacy search hits 

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2230--oak-web-application.netlify.thenational.academy
search should work as before


## Acceptance Criteria:

- [ ]  Search results will show the ‘new’ tag according to their cohort
